src/app/components/JumpTo/index.tsx

```diff
@@ -4,25 +4,22 @@ import { jsx } from '@emotion/react';
 import { ServiceContext } from '#contexts/ServiceContext';
 import useViewTracker from '#app/hooks/useViewTracker';
 import useClickTrackerHandler from '#app/hooks/useClickTrackerHandler';
-import { EventTrackingMetadata } from '#app/models/types/eventTracking';
 import Text from '#app/components/Text';
 import InlineLink from '#app/components/InlineLink';
 import idSanitiser from '../../lib/utilities/idSanitiser';
 
 export interface JumpToProps {
   jumpToHeadings?: Array<{ heading: string }>;
-  eventTrackingData?: EventTrackingMetadata;
 }
 
-const JumpTo = ({ jumpToHeadings, eventTrackingData }: JumpToProps) => {
```

Comment:
My concern with removing `eventTrackingData` as a prop probably lies beyond the use of this component for more than just an experiment. Mainly for dynamic tracking data. We have it as a prop on the `Billboard` for example. Are we okay hard-coding it then?

Reply:
It should be ok to hard code within the component itself. 

ArticlePage components do have access to additional props by default, such as `position` which we can use in the event tracking data if needed.

The Billboard component uses the event tracking data like this, to mark its position within the Homepage, but the implementation of the Homepage/Topics is quite different, so likely needed done that way.